vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewPerformanceManager.ts

```diff
@@ -82,13 +91,27 @@ export class PageViewPerformanceManager {
         }
     }
 
-    public getPerformanceTiming(): PerformanceTiming {
+    public getPerformanceTiming(): PerformanceTiming | null {
         if (this.isPerformanceTimingSupported()) {
             return window.performance.timing;
         }
 
         return null;
     }
+    public getPerformanceNavigationTiming(): PerformanceNavigationTiming | null {
+        if (this.isPerformanceNavigationTimingSupported()) {
+            return window.performance.getEntriesByType("navigation")[0] as PerformanceNavigationTiming;
```

Comment:
From what I can tell from the spec, this is always an array with a single element. Is this correct?

Reply:
Yes, there is only one `PerformanceNavigationTiming` object in the performance timeline: 
(the first green box): https://www.w3.org/TR/navigation-timing-2/#PerformanceResourceTiming
but there're 4 types of NavigationTiming:
https://www.w3.org/TR/navigation-timing-2/#sec-performance-navigation-types